pkg/check/info.go

```diff
@@ -0,0 +1,37 @@
+package check
+
+import (
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+const (
+	versionLabel = "version"
+)
+
+var (
+	vCenterInfoMetric = metrics.NewGaugeVec(
+		&metrics.GaugeOpts{
+			Name:           "vsphere_vcenter_info",
+			Help:           "Information about vSphere vCenter.",
+			StabilityLevel: metrics.ALPHA,
```

Comment:
There were talks to support multiple vcenters. Do we want to emit vcenter name as a label?

Reply:
I added vCenter UUID as a label. I don't want to show vCenter name, because it's URL in customers network and IMO it could be quite sensitive to report via telemetry.